src/sentry/search/events/fields.py

```diff
@@ -889,7 +889,15 @@ def get_function_alias(field: str) -> str:
 
 
 def get_function_alias_with_columns(function_name, columns) -> str:
-    columns = re.sub(r"[^\w]", "_", "_".join(str(col) for col in columns))
+    columns = re.sub(
+        r"[^\w]",
+        "_",
+        "_".join(
+            # Encode to ascii with backslashreplace so unicode chars become \u1234, then decode cause encode gives bytes
+            str(col).encode("ascii", errors="backslashreplace").decode()
```

Comment:
Does unicode in params affect clickhouse in any other functions, if so, does this need to be handled higher up in the request?